readme.md

```diff
@@ -168,11 +168,10 @@ erDiagram
 
    notifications {
       int id PK
+      boolean is_read
       dateTime created_at
       dateTime updated_at
       int receiver_user_id FK
-      enum status
```

Comment:
Having enums is much better than having flags. I would recommend moving it back
cc @liza-veis

Reply:
agree, let's use status enum instead